astro/src/content/quickstarts/quickstart-flutter-native.mdx

```diff
@@ -0,0 +1,288 @@
+
+In that case, the system might look like this before FusionAuth is introduced.
+
+<LoginBefore alt={"Request flow during login before FusionAuth"}/>
+
+The login flow will look like this after FusionAuth is introduced.
+
+<LoginAfter alt={"Request flow during login after FusionAuth"}/>
+
+In general, you would introduce FusionAuth to normalize and consolidate user data, making it consistent and up-to-date and offloading your login security and functionality to FusionAuth.
+
+
+## Getting Started
+
+Start with getting FusionAuth up and running and creating a Flutter application.
+
+
+### Clone The Code
+
+First, grab the code from the repository and change into that directory.
+
+```shell
+git clone https://github.com/FusionAuth/fusionauth-quickstart-flutter-native.git
+cd fusionauth-quickstart-flutter-native
+```
+
+All shell commands in this guide can be entered in a terminal in this folder. On Windows, you need to replace forward slashes with backslashes in paths.
+
+The files you'll create in this guide already exist in the `complete-application` folder, if you prefer to copy them.
+
+
+### Run FusionAuth Via Docker
+
+You'll find a Docker Compose file `docker-compose.yml` and an environment variables configuration file `.env` in the root directory of the repo.
+
+Assuming you have Docker installed, you can start FusionAuth on your machine with the following.
+
+```shell
+docker-compose up -d
+```
+
+This will start three containers, one each for FusionAuth, Postgres, and Elasticsearch.
+
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart). When FusionAuth starts for the first time, it will look at the `kickstart/kickstart.json` file and configure FusionAuth to your specified state.
+
+<Aside type="note">
+If you ever want to reset the FusionAuth system, delete the volumes created by `docker-compose` by executing `docker-compose down -v`, then rerun `docker-compose up -d`.
+</Aside>
+
+FusionAuth will be configured with these settings:
+
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth is `http://localhost:9011/`.
+
+You can log in to the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want to, but with Docker and Kickstart, everything will already be configured correctly.
+
+<Aside type="caution">
+  The `.env` and `kickstart.json` files contain passwords. In a real application, always add these files to your `.gitignore` file and never commit secrets to version control.
+</Aside>
+
+
+### Set Up A Public URL For FusionAuth
```

Comment:
Could we add some verbiage here to include this as an easy guide?
https://fusionauth.io/docs/v1/tech/developer-guide/exposing-instance

Then include this script
```shell
ngrok http --request-header-add 'X-Forwarded-Port:443' 9011
```